tests/e2e/functional_test.go

```diff
@@ -724,7 +726,7 @@ func createNumaflowControllerRolloutSpec(name, namespace string) *apiv1.Numaflow
 			Namespace: namespace,
 		},
 		Spec: apiv1.NumaflowControllerRolloutSpec{
-			Controller: apiv1.Controller{Version: "0.0.18"},
+			Controller: apiv1.Controller{Version: "1.3.3"},
```

Comment:
Maybe keep these defined as constants

Reply:
replaced these and the Jetstream versions both with constants